fiboa_cli/datasets/ec_ee.py

```diff
@@ -1,83 +1,62 @@
 import pandas as pd
 
-from ..convert_utils import convert as convert_
-from .commons.ec import add_eurocrops
+from ..convert_utils import BaseConverter
+from .commons.ec import EuroCropsConverterMixin
 
 # todo: The dataset doesn't validate due to a self intersecting polygon
 # How do we want to handle this?
 
-SOURCES = "https://zenodo.org/records/8229128/files/EE_2021.zip?download=1"
 
-ID = "ec_ee"
-SHORT_NAME = "Estonia"
-TITLE = "Field boundaries for Estonia"
-DESCRIPTION = """
-Geospatial Aid Application Estonia Agricultural parcels.
-The original dataset is provided by ARIB and obtained from the INSPIRE theme GSAA (specifically Geospaial Aid Application Estonia Agricultural parcels) through which the data layer Fields and Eco Areas (GSAA) is made available.
-The data comes from ARIB's database of agricultural parcels.
-"""
-PROVIDERS = [
-    {
-        "name": "Põllumajanduse Registrite ja Informatsiooni Amet",
-        "url": "http://data.europa.eu/88u/dataset/pria-pollud",
-        "roles": ["producer", "licensor"],
+class Convert(EuroCropsConverterMixin, BaseConverter):
```

Comment:
The old converter sets the year specifically to 2021 in the EC call, I can't find that anymore. Is it still present?

Reply:
There's an updated version of the dataset that does contain the 2021: https://zenodo.org/records/14094196

Reply:
Ah, it get's added to the geoparquet title meta data. I missed this. Just added a fix